src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -41,7 +41,7 @@ public static class WorkerRuntimeLanguageHelper
         private static readonly IDictionary<WorkerRuntime, string> workerToDefaultLanguageMap = new Dictionary<WorkerRuntime, string>
         {
             { WorkerRuntime.dotnet, Constants.Languages.CSharp },
-            { WorkerRuntime.dotnetIsolated, Constants.Languages.CSharp },
+            { WorkerRuntime.dotnetIsolated, Constants.Languages.CSharpIsolated },
```

Comment:
Does this need entries for F#?

Reply:
I see it's the defaults

Reply:
Yeah, figured that the default of C# makes the most sense